inscriptions/ISic000527.xml

```diff
@@ -190,14 +195,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AA">
+                <ab>
+                    <lb n="1"/>Diis M<supplied reason="lost">anibus</supplied>
+                    <lb n="2"/><name><expan><abbr>P</abbr><ex>ublio</ex></expan></name> <g ref="#interpunct">·</g> <name>Stationam</name> <gap reason="lost" extent="unknown" unit="character"/> <name><supplied reason="lost">Sta</supplied>
```

Comment:
i think the first name, however it is resolved after 'statio...' must be nominative, (so 'P(ublius)'), since it seems that it is the principal who set it up for himself ('et sibi') as well as for his wife and descendants. One must admit some unknowables in the mess of line 2